tests/va_profile_delete_cron_job/test_va_profile_delete_cron_job.py

```diff
@@ -0,0 +1,78 @@
+import os
+from lambda_functions.va_profile_delete_cron_job.va_profile_delete_cron_job import va_profile_delete_cron_job_handler
+from sqlalchemy import text
+
+OPT_IN_OUT = text("""SELECT va_profile_opt_in_out(:va_profile_id, :communication_item_id, :communication_channel_id, :allowed, :source_datetime);""")
```

Comment:
Don't use this; keep your unit tests isolated from other code.  Use SQLAlchemy directly to create new rows.